index.html

```diff
@@ -0,0 +1,355 @@
+                <a href="#" class="my__icons__link"><img src="images/M.png" alt="M icon" /></a>
+              </li>
+            </ul>
+            <button type="button" class="btn clr-p-500">
+              Get my resume
+            </button>
+          </div>
+          <ul class="my__views">
+            <li class="my_languages">
+              <div class="my_languages_favourite">
+                <h3 class="my_languages_favourite__title clr-n-900">
+                  Languages
+                </h3>
+                <img class="my_languages_favourite__img" src="images/arrow.png" alt="arrow icon" />
+              </div>
+              <ul class="my_languages__bottom">
+                <li class="list">
+                  <img src="images/js.png" alt="javascript icon" />
+                  <p class="my_languages__bottom__name clr-n-700">
+                    JavaScript
+                  </p>
+                </li>
+                <li class="list">
+                  <img src="images/html.png" alt="html icon" />
+                  <p class="my_languages__bottom__name clr-n-700">
+                    HTML
+                  </p>
+                </li>
+                <li class="list">
+                  <img src="images/css.png" alt="css icon" />
+                  <p class="my_languages__bottom__name clr-n-700">
+                    CSS
+                  </p>
+                </li>
+              </ul>
+            </li>
+            <li class="my__technologies">
+              <div class="my__technologies__top">
+                <h3 class="my__technologies__top__title clr-n-900">
+                  Frameworks
+                </h3>
+                <img class="my__technologies__top__img" src="images/arrow.png" alt="arrow icon" />
+              </div>
+            </li>
+            <li class="my__skills">
+              <div class="my__skills__top">
+                <h3 class="my__skills__top__title clr-n-900">
+                  Skills
+                </h3>
+                <img class="my__skills__top__img" src="images/arrow.png" alt="arrow icon" />
+              </div>
+            </li>
+          </ul>
+        </section>
+        <section class="form-section" id="reach_me">
+            <div class="div-wrap">
+             <h2 class="form_t">Contact me</h2>
+             <p class="form-paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+             <form class="reach_me" action="https://formspree.io/f/mbjeobpq" method="post">
+               <input type="text" name="name" placeholder="Oscar Loremipsum" maxlength="30" required>
+               <input type="email" name="email" placeholder="oscar@gamil.com" required>
+               <textarea name="messages" rows="10" maxlength="500" required placeholder="Enter text here"></textarea>
+               <button type="submit" name="submit">Get in touch</button>
```

Comment:
- [ ] Good job fixing the sizes of your elements. However, please also fix the size of the submit button and the alignment of its text.

  - Your button:
    ![oscar1](https://user-images.githubusercontent.com/87394385/221226175-7e818900-8dd9-4c0c-817c-f30c80842fec.JPG)

  - Figma design button:
    ![oscar2](https://user-images.githubusercontent.com/87394385/221226487-9b75b279-4e4f-40bf-b82d-a347a5cc741b.JPG)

- [ ] Please also note that the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=2v32ThOuuscVQoPs-0) specifies a margin of 24px from the form to the page margin. Kindly check that you have the correct margins to ensure a smooth UI.

Reply:
Hi @NduatiKagiri . I do not know if that view is right becasue I see other view that does not fit with your review